src/main/java/org/folio/event/dto/HoldingEventHolder.java

```diff
@@ -24,19 +25,23 @@ public class HoldingEventHolder {
   private Map<String, String> headers;
   private String tenantId;
   private String centralTenantId;
+  private RequestContext requestContext;
```

Comment:
we keep requestContext in holder because it contains proper tenantId?

Reply:
If it just fore increasing readability I think better not to store, because it can be modified potentailly in several places and would be hard to understand what is current requestCOntext points to, it is some disadvantage of the holder

Reply:
> we keep requestContext in holder because it contains proper tenantId?

They will the same `x-okapi-tenant` as on Kafka headers to request either `holdings-record` or `instance` from `mod-inventory-storage`